pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -207,6 +208,8 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(
 		Name:             routeName,
 		VirtualHosts:     virtualHosts,
 		ValidateClusters: proto.BoolFalse,
+		// Control plane validates 1mb max size. Set this on data plane too to increase from default of 4k
```

Comment:
nit: what do you mean by this `Control plane validates 1mb max size.`

Reply:
istiod validation webhook